cognite/neat/_rules/importers/_base.py

```diff
@@ -24,28 +24,34 @@ def to_rules(self) -> ReadRules[T_InputRules]:
         raise NotImplementedError()
 
     def _default_metadata(self) -> dict[str, Any]:
-        creator = "UNKNOWN"
-        with suppress(KeyError, ImportError):
-            import getpass
-
-            creator = getpass.getuser()
-
         return {
             "schema": "partial",
             "space": "neat",
             "external_id": "NeatImportedDataModel",
             "version": "0.1.0",
             "name": "Neat Imported Data Model",
-            "created": datetime.now().replace(microsecond=0).isoformat(),
-            "updated": datetime.now().replace(microsecond=0).isoformat(),
-            "creator": creator,
+            "created": self._get_current_time(),
+            "updated": self._get_current_time(),
```

Comment:
Lets call this one above and store it such that we are sure that `created` and `updated` are equal for new models
```python
        created_time = self._current_time()
        return {
            ...
            "created": created_time,
            "updated": created_time,
          ...
        }
```